<!DOCTYPE html>
<html xmlns:th="http://www.thymeleaf.org">
<body>


<!--Menu list on the left-->
<aside th:fragment="download-menus">
    <div class="main-slide">
        <img src="/images/subtit03.gif" alt="" />
    </div>

    <div class="content-indexes">
        <br />
        <br />
        <span>
          <img src="/images/icon_5.gif" width="6px" height="10px" />
          <a class="menu" th:href="@{/download/ACTG}" href="../download/actg.html">
              ACTG
          </a>
        </span>
        <br />
        <br />
        <span>
          <img src="/images/icon_5.gif" width="6px" height="10px" />
            <a class="menu" th:href="@{/download/CIFTER}" href="../download/cifter.html">
                CIFTER
            </a>
        </span>
        <br />
        <br />
        <span>
          <img src="/images/icon_5.gif" width="6px" height="10px" />
            <a class="menu" th:href="@{/download/DBond}" href="../download/dbond.html">
                DBond
            </a>
        </span>
        <br />
        <br />
        <span>
          <img src="/images/icon_5.gif" width="6px" height="10px" />
            <a class="menu" th:href="@{/download/deMix}" href="../download/demix.html">
                deMix
            </a>
        </span>
        <br />
        <br />
        <span>
          <img src="/images/icon_5.gif" width="6px" height="10px" />
            <a class="menu" th:href="@{/download/DDPSearch}" href="../download/ddp_search.html">
                DDP Search
            </a>
        </span>
        <br />
        <br />
        <span>
          <img src="/images/icon_5.gif" width="6px" height="10px" />
            <a class="menu" th:href="@{/download/NextSearch}" href="../download/next_search.html">
                Next Search
            </a>
        </span>
        <br />
        <br />
        <span>
          <img src="/images/icon_5.gif" width="6px" height="10px" />
            <a class="menu" th:href="@{/download/MODa}" href="../download/mod_a.html">
                MODa
            </a>
        </span>
        <br />
        <br />
          <img src="/images/icon_5.gif" width="6px" height="10px" />
        <a class="menu" th:href="@{/download/MutCombinator}" href="../download/mut_combinator.html">
            MutCombinator
        </a>
        </span>
        <br />
        <br />
        <span>
          <img src="/images/icon_5.gif" width="6px" height="10px" />
            <a class="menu" th:href="@{/download/USE}" href="../download/use.html">
                USE
            </a>
        </span>
    </div>
    </aside>

<!-- Main contents of the page. Defining each fragment below. -->

<!-- Fragment: ACTG -->
<article th:fragment="ACTG">
    <div class="main-article">
        <div th:replace="~{partials/template_fragment :: main-title('ACTG', '- Novel peptide mapping')}"></div>
        <div class="main-contents no-padding width-650">
            <p>
                In many proteogenomics applications, mapping peptide sequences onto gene models can be very useful because it can find new gene model ,and correct or modify annotated gene model. ACTG provides a peptide mapping allowing four structural variations: junction variation, exon skipping, frame shift and intron retention (exon extension at exon-exon junction).
            </p>
            <br />
            <div class="grayTD">
                <img src="/images/p_st1.GIF" width="4" height="9" />
                <span class="drakBR">Software</span>
            </div>
            <ul>
                <li>Requirement</li>
                <br />
                <span class="blue">
                        ACTG was implemented using JAVA programming language, and requires Java Running Environment (JRE).
                        <br />
                        If JRE is not installed, please visit
                    </span>
                <a href="http://www.oracle.com/technetwork/java/index.html" target="_blank" class="under">http://www.oracle.com/technetwork/java/index.html</a>
                <br />
                <br />
                <li>Release Note</li>
                <br />
                - <a href="ACTG_ReleaseNote.txt" class="under"> Nov 17, 2020: version 1.20</a>
                ( <a href="/request.jsp?software=ACTG" class="under">Software download</a> )
                - Nov 5, 2018: version 1.11
                <br />
            </ul>
            <div class="grayTD">
                <img src="/images/p_st1.GIF" width="4" height="9" />
                <span class="drakBR">Publication</span>
            </div>
            <p>
                <span class="indented drakBR">ACTG: novel peptide mapping onto gene models</span>
                <em class="indented">Seunghyuk Choi, Hyunwoo Kim, and Eunok Paek.</em>
                <span class="indented">Bioinformatics, 2017, 33(8), 1218-1220. <a href="https://www.ncbi.nlm.nih.gov/pubmed/28031186" target="_blank" class="under">[ PMID: 28031186 ]</a></span>
                    <a href="https://omictools.com/amino-acids-to-genome-tool">
                        <img src="/images/omictoolslogo.JPG" width="120px" height="40px">
                    </a>
            </p>

        </div>
    </div>
</article>

<!-- Fragment: CIFTER -->
<article th:fragment="CIFTER">
    <div class="main-article">
        <div class="main-title">
            <div class="title">
                <h3 class="drakBR">
                    CIFTER
                    <span>- Charge State Determination</span>
                </h3>
                <br />
            </div>
        </div>
        <div class="main-contents no-padding width-650">
            <p>
                Low-resolution mass spectrometers cannot completely resolve isotopes of multiply-charged peptides and therefore cannot reliably determine their charge states and masses. Instead, every possible charge state, commonly +2 and +3, is assumed to calculate its mass and repetitive database searches are required for each hypothetical mass. This greatly increases the overall time needed for searching candidate peptides, and also requires additional efforts to differentiate the correct match from false positives. To alleviate these problems, CIFTER is proposed to differentiate charge states of peptides obtained from low-resolution mass spectrometers.
            </p>
            <br />
            <div class="grayTD">
                <img src="/images/p_st1.GIF" width="4" height="9" />
                <span class="drakBR">Usage</span>
            </div>
            <ul>
                <li>Requirement</li>
                <br />
                <span class="blue">
                        ACTG was implemented using JAVA programming language, and requires Java Running Environment (JRE).
                        <br />
                        If JRE is not installed, please visit
                    </span>
                <a href="http://www.oracle.com/technetwork/java/index.html" target="_blank" class="under">http://www.oracle.com/technetwork/java/index.html</a>
                <br />
                <li>Release Note</li>
                <br />
                - <a href="ACTG_ReleaseNote.txt" class="under"> Nov 17, 2020: version 1.20</a>
                ( <a href="/request.jsp?software=ACTG" class="under">Software download</a> )
                - Nov 5, 2018: version 1.11
                <br />
            </ul>
            <div class="grayTD">
                <img src="/images/p_st1.GIF" width="4" height="9" />
                <span class="drakBR">Publication</span>
            </div>
            <p>
                <span class="indented drakBR">ACTG: novel peptide mapping onto gene models</span>
                <em class="indented">Seunghyuk Choi, Hyunwoo Kim, and Eunok Paek.</em>
                <span class="indented">Bioinformatics, 2017, 33(8), 1218-1220. <a href="https://www.ncbi.nlm.nih.gov/pubmed/28031186" target="_blank" class="under">[ PMID: 28031186 ]</a></span>
                <a href="https://omictools.com/amino-acids-to-genome-tool">
                    <img src="/images/omictoolslogo.JPG" width="120px" height="40px">
                </a>
            </p>

        </div>
    </div>
</article>

<!-- Fragment: DBond -->
<article th:fragment="DBond">
    <div class="main-article">
        <div th:replace="~{partials/template_fragment :: main-title('DBond', '- Identification of intact disulfide linkages')}"></div>
        <div class="main-contents no-padding width-650">
            <p>
                DBond assigns disulfide bond sites in a protein via analysis of MS/MS spectra of disulfide linked peptides from protein digestion under non-reducing condition. A disulfide linked peptide produces multiple fragment ions from two peptides, and therefore, its MS/MS spectrum shows complex patterns and includes many overlapping fragment ions. DBond was designed to recognize diagnostic fragment ions resultant specifically from disulfide linkages.
            </p>
            <br />
            <div class="grayTD">
                <img src="/images/p_st1.GIF" width="4" height="9" />
                <span class="drakBR">Software</span>
            </div>
            <ul>
                <li>Requirement</li>
                <br />
                <span class="blue">
                        DBond was implemented using JAVA programming language, and requires Java Running Environment (JRE).
                        <br />
                        If JRE is not installed, please visit
                    </span>
                <a href="http://www.oracle.com/technetwork/java/index.html" target="_blank" class="under">http://www.oracle.com/technetwork/java/index.html</a>
                <br />
                <li>Release Note</li>
                <br />
                - <span class="blue"> Oct 01, 2012: version 3.02</span>
                ( <a href="/request.jsp?software=ACTG" class="under">Software download</a> )
                - Nov 5, 2018: version 1.11
                <br />
                <li>Spectral Viewer</li>
            </ul>
            <div class="grayTD">
                <img src="/images/p_st1.GIF" width="4" height="9" />
                <span class="drakBR">Publication</span>
            </div>
            <p>
                <span class="indented drakBR">ACTG: novel peptide mapping onto gene models</span>
                <em class="indented">Seunghyuk Choi, Hyunwoo Kim, and Eunok Paek.</em>
                <span class="indented">Bioinformatics, 2017, 33(8), 1218-1220. <a href="https://www.ncbi.nlm.nih.gov/pubmed/28031186" target="_blank" class="under">[ PMID: 28031186 ]</a></span>
                <a href="https://omictools.com/amino-acids-to-genome-tool">
                    <img src="/images/omictoolslogo.JPG" width="120px" height="40px">
                </a>
            </p>

        </div>
    </div>
</article>





<!-- Fragment: deMix -->
<article th:fragment="deMix">
    <div class="main-article">
        <div class="main-title">
            <div class="title">
                <h3 class="drakBR">
                    deMix
                    <span> - Decoding deuterated Distributions from heterogeneous protein states via HDX-MS</span>
                </h3>
                <br />
            </div>
        </div>
        <div class="main-contents no-padding width-650">
            <p>
                deMix deals directly with the deuterated isotopic distributions, but not considering their centroid masses and is designed to be robust over random noises. In addition, unlike the existing approaches that can only determine a single state from an isotopic distribution, deMix can also detect a bimodal deuterated distribution, arising from EX1 behavior or heterogeneous peptides in conformational isomer proteins. Furthermore, deMix comes with visualization software to facilitate validation and representation of the analysis results.
            </p>
            <br />
            <div class="grayTD">
                <img src="/images/p_st1.GIF" width="4" height="9" />
                <span class="drakBR">Software</span>
            </div>
            <ul>
                <li>Requirement</li>
                <br />
                <span class="blue">
                        ACTG was implemented using JAVA programming language, and requires Java Running Environment (JRE).
                        <br />
                        If JRE is not installed, please visit
                    </span>
                <a href="http://www.oracle.com/technetwork/java/index.html" target="_blank" class="under">http://www.oracle.com/technetwork/java/index.html</a>
                <br />
                <li>Release Note</li>
                <br />
                - <a href="ACTG_ReleaseNote.txt" class="under"> Nov 17, 2020: version 1.20</a>
                ( <a href="/request.jsp?software=ACTG" class="under">Software download</a> )
                - Nov 5, 2018: version 1.11
                <br />
            </ul>
            <div class="grayTD">
                <img src="/images/p_st1.GIF" width="4" height="9" />
                <span class="drakBR">Publication</span>
            </div>
            <p>
                <span class="indented drakBR">ACTG: novel peptide mapping onto gene models</span>
                <em class="indented">Seunghyuk Choi, Hyunwoo Kim, and Eunok Paek.</em>
                <span class="indented">Bioinformatics, 2017, 33(8), 1218-1220. <a href="https://www.ncbi.nlm.nih.gov/pubmed/28031186" target="_blank" class="under">[ PMID: 28031186 ]</a></span>
                <a href="https://omictools.com/amino-acids-to-genome-tool">
                    <img src="/images/omictoolslogo.JPG" width="120px" height="40px">
                </a>
            </p>

        </div>
    </div>
</article>




<!-- Fragment: DDP Search -->
<article th:fragment="DDPSearch">
    <div class="main-article">
        <div class="main-title">
            <div class="title">
                <h3 class="drakBR">
                    DDP Search
                    <span> - Data-dependent scoring parameter optimization</span>
                </h3>
                <br />
            </div>
        </div>
        <div class="main-contents no-padding width-650">
            <p>
                DDP search conducts a preliminary search for the spectra selected by the spectrum quality filter. Search results from the preliminary search are used to generate data-dependent scoring parameters, then the full search over the entire input spectra are conducted using the learned scoring parameters.
            </p>
            <br />
            <div class="grayTD">
                <img src="/images/p_st1.GIF" width="4" height="9" />
                <span class="drakBR">Software</span>
            </div>
            <ul>
                <li>Requirement</li>
                <br />
                <span class="blue">
                        ACTG was implemented using JAVA programming language, and requires Java Running Environment (JRE).
                        <br />
                        If JRE is not installed, please visit
                    </span>
                <a href="http://www.oracle.com/technetwork/java/index.html" target="_blank" class="under">http://www.oracle.com/technetwork/java/index.html</a>
                <br />
                <li>Release Note</li>
                <br />
                - <a href="ACTG_ReleaseNote.txt" class="under"> Nov 17, 2020: version 1.20</a>
                ( <a href="/request.jsp?software=ACTG" class="under">Software download</a> )
                - Nov 5, 2018: version 1.11
                <br />
            </ul>
            <div class="grayTD">
                <img src="/images/p_st1.GIF" width="4" height="9" />
                <span class="drakBR">Publication</span>
            </div>
            <p>
                <span class="indented drakBR">ACTG: novel peptide mapping onto gene models</span>
                <em class="indented">Seunghyuk Choi, Hyunwoo Kim, and Eunok Paek.</em>
                <span class="indented">Bioinformatics, 2017, 33(8), 1218-1220. <a href="https://www.ncbi.nlm.nih.gov/pubmed/28031186" target="_blank" class="under">[ PMID: 28031186 ]</a></span>
                <a href="https://omictools.com/amino-acids-to-genome-tool">
                    <img src="/images/omictoolslogo.JPG" width="120px" height="40px">
                </a>
            </p>

        </div>
    </div>
</article>



<!-- Fragment: Next Search -->
<article th:fragment="NextSearch">
    <div class="main-article">
        <div class="main-title">
            <div class="title">
                <h3 class="drakBR">
                    Next Search
                    <span>- Exon Graph Search</span>
                </h3>
                <br />
            </div>
        </div>
        <div class="main-contents no-padding width-650">
            <p>
                NextSearch includes modules (1) that constructs a compact nucleotide exon graph while incorporating novel splice variations (2) that identifies peptides by searching tandem mass spectra against the nucleotide exon graph built during the previous module.
            </p>
            <br />
            <div class="grayTD">
                <img src="/images/p_st1.GIF" width="4" height="9" />
                <span class="drakBR">Software</span>
            </div>
            <ul>
                <li>Requirement</li>
                <br />
                <span class="blue">
                        ACTG was implemented using JAVA programming language, and requires Java Running Environment (JRE).
                        <br />
                        If JRE is not installed, please visit
                    </span>
                <a href="http://www.oracle.com/technetwork/java/index.html" target="_blank" class="under">http://www.oracle.com/technetwork/java/index.html</a>
                <br />
                <li>Release Note</li>
                <br />
                - <a href="ACTG_ReleaseNote.txt" class="under"> Nov 17, 2020: version 1.20</a>
                ( <a href="/request.jsp?software=ACTG" class="under">Software download</a> )
                - Nov 5, 2018: version 1.11
                <br />
            </ul>
            <div class="grayTD">
                <img src="/images/p_st1.GIF" width="4" height="9" />
                <span class="drakBR">Publication</span>
            </div>
            <p>
                <span class="indented drakBR">ACTG: novel peptide mapping onto gene models</span>
                <em class="indented">Seunghyuk Choi, Hyunwoo Kim, and Eunok Paek.</em>
                <span class="indented">Bioinformatics, 2017, 33(8), 1218-1220. <a href="https://www.ncbi.nlm.nih.gov/pubmed/28031186" target="_blank" class="under">[ PMID: 28031186 ]</a></span>
                <a href="https://omictools.com/amino-acids-to-genome-tool">
                    <img src="/images/omictoolslogo.JPG" width="120px" height="40px">
                </a>
            </p>

        </div>
    </div>
</article>





<!-- Fragment: MODa -->
<article th:fragment="MODa">
    <div class="main-article">
        <div class="main-title">
            <div class="title">
                <h3 class="drakBR">
                    MODa
                    <span>- Multi-blind modification search</span>
                </h3>
                <br />
            </div>
        </div>
        <div class="main-contents no-padding width-650">
            <p>
                With great biological interest in post-translational modifications (PTMs), various approaches have been introduced to identify PTMs using MS/MS. MODa is a novel "multi-blind" spectral alignment algorithm that allows for fast unrestrictive PTM searches with no limitation on the number of modifications per peptide while featuring over an order of magnitude speedup in relation to existing approaches.
            </p>
            <br />
            <div class="grayTD">
                <img src="/images/p_st1.GIF" width="4" height="9" />
                <span class="drakBR">Software</span>
            </div>
            <ul>
                <li>Requirement</li>
                <br />
                <span class="blue">
                        ACTG was implemented using JAVA programming language, and requires Java Running Environment (JRE).
                        <br />
                        If JRE is not installed, please visit
                    </span>
                <a href="http://www.oracle.com/technetwork/java/index.html" target="_blank" class="under">http://www.oracle.com/technetwork/java/index.html</a>
                <br />
                <li>Release Note</li>
                <br />
                - <a href="ACTG_ReleaseNote.txt" class="under"> Nov 17, 2020: version 1.20</a>
                ( <a href="/request.jsp?software=ACTG" class="under">Software download</a> )
                - Nov 5, 2018: version 1.11
                <br />
            </ul>
            <div class="grayTD">
                <img src="/images/p_st1.GIF" width="4" height="9" />
                <span class="drakBR">Publication</span>
            </div>
            <p>
                <span class="indented drakBR">ACTG: novel peptide mapping onto gene models</span>
                <em class="indented">Seunghyuk Choi, Hyunwoo Kim, and Eunok Paek.</em>
                <span class="indented">Bioinformatics, 2017, 33(8), 1218-1220. <a href="https://www.ncbi.nlm.nih.gov/pubmed/28031186" target="_blank" class="under">[ PMID: 28031186 ]</a></span>
                <a href="https://omictools.com/amino-acids-to-genome-tool">
                    <img src="/images/omictoolslogo.JPG" width="120px" height="40px">
                </a>
            </p>

        </div>
    </div>
</article>





<!-- Fragment: MutCombinator -->
<article th:fragment="MutCombinator">
    <div class="main-article">
        <div class="main-title">
            <div class="title">
                <h3 class="drakBR">
                    MutCombinator
                    <span>- Combinatorial mutation search</span>
                </h3>
                <br />
            </div>
        </div>
        <div class="main-contents no-padding width-650">
            <p>
                MutCombinator enables us to practically identify mutated peptides from tandem mass spectra allowing combinatorial mutations during the database search. It uses an upgraded version of a variant graph, keeping track of frame information. The variant graph is indexed by nine nucleotides for fast access. Combinations of point mutations are considered, and also it can be practically applied together with a large mutation database such as COSMIC. Furthermore, MutCombinator supports in-frame search for coding regions and three-frame search for noncoding regions.
            </p>
            <br />
            <div class="grayTD">
                <img src="/images/p_st1.GIF" width="4" height="9" />
                <span class="drakBR">Software</span>
            </div>
            <ul>
                <li>Requirement</li>
                <br />
                <span class="blue">
                        ACTG was implemented using JAVA programming language, and requires Java Running Environment (JRE).
                        <br />
                        If JRE is not installed, please visit
                    </span>
                <a href="http://www.oracle.com/technetwork/java/index.html" target="_blank" class="under">http://www.oracle.com/technetwork/java/index.html</a>
                <br />
                <li>Release Note</li>
                <br />
                - <a href="ACTG_ReleaseNote.txt" class="under"> Nov 17, 2020: version 1.20</a>
                ( <a href="/request.jsp?software=ACTG" class="under">Software download</a> )
                - Nov 5, 2018: version 1.11
                <br />
            </ul>
            <div class="grayTD">
                <img src="/images/p_st1.GIF" width="4" height="9" />
                <span class="drakBR">Publication</span>
            </div>
            <p>
                <span class="indented drakBR">ACTG: novel peptide mapping onto gene models</span>
                <em class="indented">Seunghyuk Choi, Hyunwoo Kim, and Eunok Paek.</em>
                <span class="indented">Bioinformatics, 2017, 33(8), 1218-1220. <a href="https://www.ncbi.nlm.nih.gov/pubmed/28031186" target="_blank" class="under">[ PMID: 28031186 ]</a></span>
                <a href="https://omictools.com/amino-acids-to-genome-tool">
                    <img src="/images/omictoolslogo.JPG" width="120px" height="40px">
                </a>
            </p>

        </div>
    </div>
</article>





<!-- Fragment: USE -->
<article th:fragment="USE">
    <div class="main-article">
        <div class="main-title">
            <div class="title">
                <h3 class="drakBR">
                    USE
                    <span>-  - for MOD<sup>i</sup> and DBond</span>
                </h3>
                <br />
            </div>
        </div>
        <div class="main-contents no-padding width-650">
            <div class="grayTD">
                <img src="/images/p_st1.GIF" width="4" height="9" />
                <span class="drakBR">Software</span>
            </div>
            <ul>
                <li>Requirement</li>
                <br />
                <span class="blue">
                        ACTG was implemented using JAVA programming language, and requires Java Running Environment (JRE).
                        <br />
                        If JRE is not installed, please visit
                    </span>
                <a href="http://www.oracle.com/technetwork/java/index.html" target="_blank" class="under">http://www.oracle.com/technetwork/java/index.html</a>
                <br />
                <li>Release Note</li>
                <br />
                - <a href="ACTG_ReleaseNote.txt" class="under"> Nov 17, 2020: version 1.20</a>
                ( <a href="/request.jsp?software=ACTG" class="under">Software download</a> )
                - Nov 5, 2018: version 1.11
                <br />
            </ul>
            <div class="grayTD">
                <img src="/images/p_st1.GIF" width="4" height="9" />
                <span class="drakBR">Publication</span>
            </div>
            <p>
                <span class="indented drakBR">ACTG: novel peptide mapping onto gene models</span>
                <em class="indented">Seunghyuk Choi, Hyunwoo Kim, and Eunok Paek.</em>
                <span class="indented">Bioinformatics, 2017, 33(8), 1218-1220. <a href="https://www.ncbi.nlm.nih.gov/pubmed/28031186" target="_blank" class="under">[ PMID: 28031186 ]</a></span>
                <a href="https://omictools.com/amino-acids-to-genome-tool">
                    <img src="/images/omictoolslogo.JPG" width="120px" height="40px">
                </a>
            </p>

        </div>
    </div>
</article>

</body>
</html>
